src/components/newnav.js

```diff
@@ -56,13 +62,25 @@ function Newnav({ transparent }) {
               alt="Empathy Bytes Logo"
             />
             </Link>
-            <h1 className="nav-title"> Empathy Bytes</h1>
+            <h2 className="body"> Empathy Bytes</h2>
```

Comment:
Pr looks great! Only thing is, I think i'd be great if users could not only click on the logo, but also the name of our VIP to return to the hime page. This should be as easy as wrapping the h2 in a link as well. 

```
<Link to={"/"}>
            <h2 className="body"> Empathy Bytes</h2>
 </Link>
```